resolwe_bio/processes/alignment/hisat2.yml

```diff
@@ -58,13 +74,32 @@
 
       {% if reads|type|subtype('data:reads:fastq:single:') %}
 
-        hisat2 -p {{threads}} --no-unal -x ${INDEX} -U ${FW_READS} -S alignment_unsorted.sam 2> stats.txt
+        hisat2 \
+          -p {{threads}} \
+          --no-unal \
+          -x ${INDEX} \
+          -U ${FW_READS} \
+          -S alignment_unsorted.sam \
+          {% if noncansplice %}--pen-noncansplice {{noncansplice}}{% endif %} \
+          {% if softclip %}--no-softclip{% endif %} \
+          {% if cufflinks %}--dta-cufflinks{% endif %} \
+          2> stats.txt
         re-checkrc "HISAT2 alignment failed."
         re-progress 0.6
 
       {% else %}
 
-        hisat2 -p {{threads}} --no-unal -x ${INDEX} -1 ${FW_READS} -2 ${RW_READS} -S alignment_unsorted.sam 2> stats.txt
+        hisat2 \
+          -p {{threads}} \
+          --no-unal \
+          -x ${INDEX} \
+          -1 ${FW_READS} \
+          -2 ${RW_READS} \
```

Comment:
This is the only line that is different in two commands. Please combine them to one and use if for this line.